src/main/java/nl/tudelft/ewi/devhub/server/database/entities/notifications/Notification.java

```diff
@@ -0,0 +1,51 @@
+package nl.tudelft.ewi.devhub.server.database.entities.notifications;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import nl.tudelft.ewi.devhub.server.database.entities.User;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.validator.constraints.NotEmpty;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.net.URI;
+import java.util.Date;
+
+/**
+ * Created by Arjan on 7-6-2017.
+ */
+@Data
+@Entity
+@Table(name = "notification")
+@EqualsAndHashCode(of = {"id"})
+public class Notification {
+
+    @Id
+    @Column(name = "id")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @Column(name = "event")
+    @NotNull
+    //Change this to Enum?
+    private String event;
+
+    @Column(name = "message")
+    @NotEmpty
+    private String message;
+
+    @Column(name = "link")
+    // Change this to URI later
+    private String link;
```

Comment:
What I had in mind is:
```JAVA
public interface Viewable {
   public URI getUri()
}
```
This interface is then implemented by `PullRequest`, `Issue` and other classes for Notifications.
Then in `Notification` you get something like this:
```JAVA
@Column(name = "linkedViewable")
private Viewable linkedViewable

public URI getUri() {
    return Viewable.getUri();
}
```
And then in the database there is one field for the `ID` of the `Viewable` and one field for the `type` since different types can share the same ID. Is it possible to map this to different entities using hibernarnate?

When this is not the case there is still to possibility to use it in this way: 
```JAVA
@Column(name = "viewableId")
private int viewable;
@Colum(name = "type")
private Enum type

public URI getUri() {
    return getViewableObject().getUri();
}

public getViewableObject() {
    switch(type)
         case PULL_REQUEST:
                return query().from(pullRequests).where(pullRequest.id.equals(viewable));
         case ISSUE:
        // query for other viewable objects.
}
```
has the down side that viewable needs to be retrieved every time from the database the URI is requested. In this case using subtypes is probably a better idea.